brainstormforce/ultimate-addons-for-gutenberg
# Fix: Marketing button - Added hide/show description option and V-align  issue.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/cccf580c-482a-4990-8576-b35bfbbaf352?redirect=true&view=06c01065-22db-4b44-9a83-f0eceb06e66a&t=attributes

### Screenshots

- https://share.getcloudapp.com/nOuR58vA

### Types of changes




- Bug

### How has this been tested?

- Drag and drop marketing buttons.
- Check the V-align of the icon with the respective title font size.
- Check Icon color same as the title or not. Icon should pull color from the theme styles.
- Check the Show/Hide option for Description.

### Checklist:
